tests/Integration/inc/Engine/CriticalPath/APIClient/sendGenerationRequest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\Engine\CriticalPath\APIClient;
+
+use WP_Rocket\Engine\CriticalPath\APIClient;
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+use Brain\Monkey\Functions;
+use WP_Error;
+
+/**
+ * @covers \WP_Rocket\Engine\CriticalPath\APIClient::send_generation_request
+ * @group CriticalPath
+ * @group  vfs
+ */
+class Test_SendGenerationRequest extends FilesystemTestCase {
+
+	protected $path_to_test_data = '/inc/Engine/CriticalPath/APIClient/sendGenerationRequest.php';
```

Comment:
Is the filesystem used in the `APIClient::send_generation_request() method or its dependencies? 

If no:

1. Switch this test case to `TestCase`
2. Remove the property `$path_to_test_data`
3. Change the data provider to `configTestData`
4. Remove the vfs and `test_data` from the test data fixture, i.e. make it an array of test data